google/cloud/aiplatform/experimental/vertex_model/serializers/pandas.py

```diff
@@ -29,7 +30,9 @@
     )
 
 
-def _serialize_dataframe(artifact_uri: str, obj: pd.DataFrame, temp_dir: str) -> str:
+def _serialize_dataframe(
+    artifact_uri: str, obj: pd.DataFrame, dataset_type: str
```

Comment:
dataset_type seems like it can be removed since we know the object is a `pd.Dataframe`

Reply:
dataset_type refers to training, testing, etc.

Reply:
I see, that makes sense. May want to rename this `parameter_name` to be consistent with the usage in the source maker.